modules/Surveys/Entry/Survey.php

```diff
@@ -378,13 +378,17 @@ function displayClosedPage($survey)
     <div class="container">
```

Comment:
Hi @serhiisamko091184, also, maybe better to have this full html code on a template instead of doing it by breaking the php tags.